app/src/ui/changes/changed-file.tsx

```diff
@@ -100,6 +101,15 @@ export class ChangedFile extends React.Component<IChangedFileProps, void> {
       })
     }
 
+    if (this.props.status !== FileStatus.Deleted) {
```

Comment:
Could we still display this item but set it as disabled when the file is not around on disk?